erts/autoconf/vendor.info

```diff
@@ -1,5 +1,10 @@
+%% %CopyrightBegin%
+%%
 %% SPDX-License-Identifier: Apache-2.0
-%% SPDX-FileCopyrightText: 2024 Erlang/OTP and its contributors
+%%
+%% Copyright Ericsson AB 2025. All Rights Reserved.
```

Comment:
same here, I think it is right that the year was added in 2025

Reply:
I changed the year of all vendor.info to be 2025 as that is when they were added.